plugins/org.jkiss.dbeaver.ext.postgresql/src/org/jkiss/dbeaver/ext/postgresql/model/PostgreProcedure.java

```diff
@@ -311,6 +319,22 @@ public void setKind(PostgreProcedureKind kind) {
     public long getObjectId() {
         return oid;
     }
+    
+    public long getXmin() {
+        return xmin;
+    }
+    
+    public boolean getProcWasCreatedOrReplaced() {
+        return procWasCreatedOrReplaced;
+    }
+    
+    public void setProcWasCreatedOrReplaced(boolean value) {
+        this.procWasCreatedOrReplaced = value;
+    }
+    
+    public boolean getProcWasNotChanged() {
+        return procWasNotChanged;
+    }    
```

Comment:
Not entirely sure that we need two variables to mark that proc was changed. At least I don't see `procWasNotChanged` checked anywhere.